src/AzureAuth/Commands/Ado/CommandPat.cs

```diff
@@ -168,15 +174,22 @@ private bool ValidOptions(ILogger logger)
             }
             else
             {
-                var invalidScopes = AdoPat.Scopes.Validate(this.Scopes);
-                if (!invalidScopes.IsEmpty)
+                if(NoScopeValidation)
+                {
+                    logger.LogWarning($"Received {NoValidationFlag} flag. AzureAuth will skip validation of scopes.");

```

Comment:
Could also simplify this message to "Skipping scopes validation"

Reply:
Consider moving this nested check into the outer if/else block so you have 3 main branches:
1. no scopes, print error, return false
2. there are scopes, but skipValidation is passed, warn, do nothing else,
3. do scope validation

Reply:
Done! Thanks.